client/packages/invoices/src/Returns/api/api.ts

```diff
@@ -3,14 +3,22 @@ import { Sdk } from './operations.generated';
 
 export const getReturnsQueries = (sdk: Sdk, storeId: string) => ({
   get: {
-    newSupplierReturnLines: async (lineIds: string[]) => {
+    newSupplierReturnLines: async (inboundShipmentLineIds: string[]) => {
       const result = await sdk.newSupplierReturnLines({
-        inboundShipmentLineIds: lineIds,
+        inboundShipmentLineIds,
         storeId,
       });
 
       return result?.newSupplierReturn;
     },
+    inboundReturnLines: async (outboundShipmentLineIds: string[]) => {
```

Comment:
Hmm, i guess you are coordinating with Carl about this one, since invoiceByNumber is needed for detail view ?